docs/getting_started.rst

```diff
@@ -4,16 +4,34 @@ Getting Started
 =====================
 
 Welcome to Fast-Pauli from `Qognitive <https://www.qognitive.io/>`_, an open-source Python / C++ library for optimized operations on Pauli matrices and Pauli strings
-based on `PauliComposer <https://arxiv.org/abs/2301.00560>`_. In this guide,
-we'll introduce some of the important operations to help users get started. For more details,
-see the API documentation.
+based on `PauliComposer <https://arxiv.org/abs/2301.00560>`_.
+In this guide, we'll introduce some of the important operations to help users get started as well as some conceptual background on Pauli matrices and Pauli strings.
+
+For more details, see the :doc:`python_api` or :doc:`cpp_api` documentation.
 
 For tips on installing the library, check out the guide: :doc:`index`.
 
 Pauli Matrices
 ------------------------
 
-For a conceptual overview of Pauli matrices, see `here <https://en.wikipedia.org/wiki/Pauli_matrices>`_.
+For a more in-depth overview of Pauli matrices, see `here <https://en.wikipedia.org/wiki/Pauli_matrices>`_.
+
+In math and physics, a `Pauli matrix <https://en.wikipedia.org/wiki/Pauli_matrices>`_, named after the physicist Wolfgang Pauli, is any one of the special 2 x 2 complex matrices in the set (often denoted by the greek letter :math:`\sigma`) :
+
+.. math::
+
+    \sigma_0 = \begin{bmatrix} 1 & 0 \\ 0 & 1 \end{bmatrix}
```

Comment:
technically not a pauli matrix

Reply:
Changed to just include Sigma_X,Y,Z and then mention I=Sigma_0 after